test/recipes.spec.js

```diff
@@ -140,5 +143,66 @@ describe('Recipes', () => {
           done(err);
         });
     });
+
+    it('should update recipe', done => {
+      chai
+        .request(server)
+        .put(`${baseUrl}/${slug}`)
+        .set({ authorization: token })
+        .send({ preparationTime: 5000 })
+        .end((err, res) => {
+          const { body } = res;
+          expect(res).to.have.status(200);
+          expect(body.recipe).to.be.an('object');
+          expect(body.recipe.id).to.be.a('number');
+          expect(body.recipe).to.haveOwnProperty('title');
```

Comment:
It would be better to test that these actually match the expected state of the recipe after updating.

Reply:
Alright. noted

Reply:
done